### ISSUE TITLE
Handle dates other than due date

### ISSUE THREAD

**Issue by roovo** [roovo/obsidian-card-board#94]
Should I consider supporting things like start, scheduled, do (whatever these are) dates?

**Comment by roovo** [comment]
For the Tasks plugin, there are [due, scheduled, and start dates](https://obsidian-tasks-group.github.io/obsidian-tasks/getting-started/dates/).

Looks like the most appropriate interpretation would be to use the scheduled date to determine the column on dated boards if present.  If not,  use the due date.  I don't think Start dates are relevant for CardBoard.

**Comment by wimlep** [comment]
would also like this scheduled date support as to be able to use cardboard view for scheduling

**Comment by roovo** [comment]
There is also the [Reminder](https://obsidian-reminder.cf/) plugin which has reminder dates as well as some compatibility with the [Tasks](https://obsidian-tasks-group.github.io/obsidian-tasks/) plugin.

**Comment by roovo** [comment]
[@ItsFullOfCode](https://github.com/ItsFullOfCode) kicked off a discussion (#110) on excluding part of task's description from task text - mentioning Reminders timestamps in particular.  Once I support parsing other types of dates, I need to think about how to display them on a card as they do not belong in the displayed title.

**Comment by roovo** [comment]
See: https://github.com/roovo/obsidian-card-board/issues/93#issuecomment-1415232770
[@DerNamenlose]
I'd like to add my 2 cents and propose to use the Start date from the Tasks plugin to hide tasks from the board. I at least use the Start date as a kind of "don't bother me until then" marker.

**Comment by Bjaux** [comment]
I second the comment #93 , I too use a start date or hide until date a lot for things that I just cannot start with until date X and for that don't want to be bothered with. This would help alot!

**Comment by fl0wm0ti0n** [comment]
Yes i Need this! I just wanted to ask for this, I normally order my tasks after scheduled and not after due dates. So I will need this

best would if if i can use due or the scheduled date for each individual column i set up and also a the "recurring" column would be nice

working with the Task plugin!

**Comment by roovo** [comment]
My attempt at a summary to check my understanding on this area:

There are 3 date types which seem to be particularly relevant (definitions as per Tasks plugin):
- Due (date by when a task must be done)
- Scheduled (when you plan to work on it)
- Start (date before which you can't start)

Some other date types are:
- Cancelled date - this would be looked at as part of dealing with custom status's (alternative checkboxes) - see #171.
- Created Date
- Reminder date/time (reminder plugin)

Proposed behaviour:
- If a  task has a start date then exclude it from columns if the date is prior to the start date.
- Allow the definition of a column for tasks which have not yet reached their start date -> so they can be accessed.
- The date used to determine which column a task should appear in will be: scheduled date (if present), due date (if there is no scheduled date), start date (if neither due or scheduled date is given).
- The task will be considered undated if it has none of these 3 dates, so would be shown in the Undated column if present.
- The file date (where use of file dates is enabled) will be considered as if it is a scheduled date.
- None of these dates (or cancelled, created, reminder dates) will appear in the title of the card.

Not yet thought about: which date(s) should be shown on the card and where should they go?

**Comment by heeeedjr** [comment]
Hi, I love the plugin! It has been so helpful for my workflow. I was just wondering if there is any update on allowing for scheduled dates as mentioned here. I imagine you have a lot going on so no worries just a friendly inquiry. Thanks!!
